src/react/components/pages/editorPage/editorPage.scss

```diff
@@ -144,14 +148,43 @@
     height: 100%;
     width: 100%;
     z-index: 0;
-    background-color: $lighter-3;
 }
 
 .full-size {
     width: 100%;
     height: 100%;
 }
 
-#editor-zone svg {
-    box-shadow: 0px 0px 0.25px #999, 0px 0px 10px rgb(0,0,0);
+.canvas-enabled {
+    .video-react {
+        z-index: 0;
+    }
+}
+
+.canvas-disabled {
+    .video-react {
+        z-index: 1;
+    }
+}
+
+canvas {
+    background-color: #1e2024 !important;
+}
+
+.video-react {
```

Comment:
Override video controls specifically not to overlap the video element so we don't have to deal with selection/editor zone falling on top of the video controls.